src/main/java/network/brightspots/rcv/Logger.java

```diff
@@ -98,14 +98,24 @@ static void setup() {
   }
 
   // adds file logging for a tabulation run
-  static void addTabulationFileLogging(String outputPath) throws IOException {
-    // use Level.FINE to capture audit info
-    tabulationHandler =
-        new FileHandler(outputPath, LOG_FILE_MAX_SIZE_BYTES, TABULATION_LOG_FILE_COUNT, true);
+  static void addTabulationFileLogging(String outputFolder, String timestampString)
+      throws IOException {
+    // log file name is: outputFolder + timestamp + log index
+    // FileHandler requires % to be encoded as %%.  %g is the log index
+    String tabulationLogPattern =
+            Paths.get(outputFolder.replace("%", "%%"),
+                    String.format("%s_audit_%%g.log", timestampString))
+                    .toAbsolutePath()
+                    .toString();
+
+    tabulationHandler = new FileHandler(tabulationLogPattern,
+            LOG_FILE_MAX_SIZE_BYTES,
+            TABULATION_LOG_FILE_COUNT,
+            true);
     tabulationHandler.setFormatter(formatter);
     tabulationHandler.setLevel(Level.FINE);
     logger.addHandler(tabulationHandler);
-    info("Tabulation logging to: %s", outputPath.replace("%g", "*"));
+    info("Tabulation logging to: %s", tabulationLogPattern.replace("%g", "0"));
```

Comment:
Why this change? It seems like it makes it look like it's always pointing to log index 0 when that may not actually be the case.

Reply:
I believe this is an improvement over the current message.  At the time it's printed it's accurate and it is less confusing than the current asterisk message, which might mean something to folks acquainted with the notion of wildcards, but not to our users.

Reply:
I guess this begs the question of why we even include the log index at all if we know it's always going to be 0? I assume it serves some purpose and, if so, I think we should keep this as-is. We've already got an established precedent of using * in a log name for the execution log message, and I think it's a pretty well-understood concept.

In any case, this PR looks good, thanks for fixing this!